db/migrate/20160403055927_create_deployment_authentications.rb

```diff
@@ -0,0 +1,33 @@
+class CreateDeploymentAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :deployment_authentications do |t|
```

Comment:
What about the `BasicAuthPasswordIdentityProvider`?  It has a url, ca, and client cert/key combination.